components/collapse/collapseItem.js

```diff
@@ -71,6 +73,10 @@ CollapseItem.propTypes = {
    * The CollapseItem's title
    */
   title: PropTypes.node.isRequired,
+  /**
+   * Specify props for label
+   */
+  labelProps: PropTypes.object,
```

Comment:
This is a little bit vague... no? Allowing everything to be passed 🙄 ... Oh well, your call.